apps/site/lib/site_web/controllers/stop_controller.ex

```diff
@@ -494,18 +494,4 @@ defmodule SiteWeb.StopController do
 
     %{conn | assigns: assigns}
   end
-
-  def get_facilities(conn, %{"stop_id" => stop_id}) do
```

Comment:
removing this breaks one of the unit tests, can you please remove that test too?